src/main/java/org/fundacionjala/coding/jorge/FizzBuzz.java

```diff
@@ -0,0 +1,59 @@
+package org.fundacionjala.coding.jorge;
+
+/**
+ *
+ */
+public final class FizzBuzz {
+    private static final int NUMBER_5 = 5;
+    private static final int NUMBER_3 = 3;
+    private static final int NUMBER_10 = 10;
+
+    /**
+     *
+     */
+    public FizzBuzz() {
+    }
+
+    /**
+     * @param rango int[]
+     * @return Object[]
+     */
+    public Object[] fizzBuzz(final int[] rango) {
+        return basic(rango);
+    }
+
+    /**
+     * The Two Parts.
+     *
+     * @param rango int[].
+     * @return arrayObj2 Object[].
+     */
+    public Object[] fizzBuzz2(final int[] rango) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 22 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)